sentry-android-fragment/src/main/java/io/sentry/android/fragment/FragmentLifecycleIntegration.kt

```diff
@@ -9,18 +9,35 @@ import io.sentry.IHub
 import io.sentry.Integration
 import io.sentry.SentryLevel.DEBUG
 import io.sentry.SentryOptions
+import io.sentry.android.fragment.internal.SentryFragmentLifecycleCallbacks
 import java.io.Closeable
 
 class FragmentLifecycleIntegration(
     private val application: Application,
-    private val enableFragmentLifecycleBreadcrumbs: Boolean,
+    private val loggedFragmentLifecycleStates: Set<FragmentLifecycleState>,
```

Comment:
how about we rename it to something more descriptive, e.g. `createFragmentLifecycleBreadcrumbsFor` or something similar. 

I think it's important to keep the `Breadcrumb` part in the variable name, because `logged` can be confusing, as in, we are logging it with an internal logger.

Reply:
Good point. Imo the best possible name would be `enabledFragmentLifecycleBreadcrumbs` (note the `d` in `enabled`). But this name is too similar to the existing `enableFragmentLifecycleBreadcrumbs`, so we must pick something else. What do you think about `filterFragmentLifecycleBreadcrumbs`?